src/scripts/http/http.ts

```diff
@@ -0,0 +1,50 @@
+/// <reference path="../../../node_modules/onenoteapi/target/oneNoteApi.d.ts" />
+
+interface ResponsePackage {
+	parsedResponse: string;
+	request: XMLHttpRequest;
+}
+
+/**
+ * Helper class for performing http requests. This is a WIP and has barebones functionality, but
```

Comment:
Currently, we only implement GET here because it's the only thing we use from what I can see. I'll double check this.